lib/Doctrine/ORM/Proxy/ProxyFactory.php

```diff
@@ -98,6 +86,9 @@ public function __serialize(): array
      */
     private $identifierFlattener;
 
+    /** @var ProxyDefinition[] */
```

Comment:
this looks like `array<class-string, ProxyDefinition>`

Reply:
I borrowed from the parent class ;)